blueocean-rest-impl/src/test/java/io/jenkins/blueocean/service/embedded/PipelineApiTest.java

```diff
@@ -797,13 +799,13 @@ public void parameterizedFreestyleTestWithoutDefaultParam() throws Exception {
                 ), 400);
     }
 
+    @Ignore
```

Comment:
@imeredith I believe this test exists to ensure that maven modules in a maven build are not flattened out to show up as separate pipelines (and that the job only shows up once in listing, and cliking goes to classic). 

So this may be a legit 🐛  that it is uncovering, or the test needs to be corrected, or at LEAST need to manually confirm we haven't regressed maven module support